src/Channel.ts

```diff
@@ -96,10 +96,14 @@ export default class Channel {
 
   public setRawDataRange(min: number, max: number): void {
     // remap the lut which was based on rawMin and rawMax to new min and max
-    const newLut = remapLut(this.lut, this.rawMin, this.rawMax, min, max);
+    // If either of the min/max ranges are both zero, then we have undefined behavior and should
+    // not remap the lut.  This situation can happen at first load, for example,
+    // when one channel has arrived but others haven't.
+    if (!(this.rawMin === 0 && this.rawMax === 0) && !(min === 0 && max === 0)) {
+      this.lut.remapDomains(this.rawMin, this.rawMax, min, max);
+    }
```

Comment:
Should this check for `this.rawMin === this.rawMax` or `min === max` instead of zero values? I think from digging through the code this would cause a divide-by-zero in `Lut.remapDomain`.

Reply:
both being zero is a possible indicator that no data has arrived yet. but nonzero and the same means all pixels are only that intensity value. So probably should handle the div by zero situation by adding 1 or something.  I will post this as an issue for a subsequent PR.